torch/csrc/jit/tracer.h

```diff
@@ -128,10 +96,10 @@ inline Value* getOutputTrace(
     size_t output_no) {
   if (!var.defined()) {
     Node* n = state->graph->createUndefined();
-    return state->graph->appendNode(n)->output();
+    return state->graph->insertNode(n)->output();
   }
 
-  auto& value_map = getTracingState()->value_map;
+  auto& value_map = getTracingState()->env_stack.back().value_map;
```

Comment:
output_no is now no longer to sufficient to accurately describe where a problem in getOutputTrace occurs. So we should either remove it (because it will be wrong and misleading) or improve the message.

Reply:
Note that only onnx tracing will ever produce multiple outputs. so output_no is almost always 0